src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/AuthenticationHelper.cs

```diff
@@ -159,6 +159,7 @@ private static void SetBasicAuthToken(HttpRequestMessage request, NetworkCredent
             // Any errors in obtaining parameter return false and we don't proceed with auth
             if (string.IsNullOrEmpty(parameter))
             {
+                if (NetEventSource.IsEnabled) NetEventSource.Error(null, "Null parameter");
```

Comment:
Should this be Info instead of Error?  I don't remember how we used Error in the past, but I thought it was more for exceptional conditions.  If this is consistent with what we do elsewhere, then no problem.

Reply:
"Error" was meant for runtime errors and not just for "exceptional conditions".  I've noticed that we have not been using this logging type in enough places.  It is emitted as a separate logging type and it's useful for developers to be able to see it separately and/or filter by it.

Reply:
Also, can we pass in some kind of context object here, like `request`?

Reply:
I removed this line and moved the logging as per Geoff's feedback https://github.com/dotnet/corefx/pull/28943#discussion_r180227016